tests/base/db/BaseActiveRecord.php

```
@@ -1321,6 +1322,39 @@ public function testDefaultValues(): void
         $this->assertEquals(['int_col2' => 1], $model->toArray());
     }
 
+    public function testLoadDefaultValuesForCompositePrimaryKey(): void
+    {
+        $db = $this->getConnection(false);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)